setup.py

```diff
@@ -37,8 +37,8 @@
     version=version,
     description="Rev.ai makes speech applications easy to build!",
     long_description=readme + '\n\n' + history,
-    author="Joshua Dong",
-    author_email='jdong42@gmail.com',
+    author="Kyle Bridburg",
+    author_email='kyle@rev.com',
```

Comment:
Should create an email alias? Example: revai-devs at rev.com.

We have a few existing aliases that we can use.

Reply:
I'll ping you separately on slack about this.